src/components/adherence/summary/AdherenceAlerts.tsx

```diff
@@ -1,143 +1,212 @@
-import {useAdherenceAlerts} from '@components/studies/adherenceHooks'
+import {useAdherenceAlerts, useUpdateAdherenceAlerts} from '@components/studies/adherenceHooks'
 import Loader from '@components/widgets/Loader'
-import CommentIcon from '@mui/icons-material/Comment'
-import {Box, Checkbox, styled} from '@mui/material'
-import IconButton from '@mui/material/IconButton'
-import List from '@mui/material/List'
-import ListItem from '@mui/material/ListItem'
-import ListItemButton from '@mui/material/ListItemButton'
-import ListItemIcon from '@mui/material/ListItemIcon'
-import ListItemText from '@mui/material/ListItemText'
-import {latoFont} from '@style/theme'
+import TablePagination from '@components/widgets/pagination/TablePagination'
+import {
+  Box,
+  Checkbox,
+  FormControlLabel,
+  FormGroup,
+  Table,
+  TableRow,
+  TableBody,
+  IconButton,
+  Typography,
+  Avatar,
+  styled,
+} from '@mui/material'
+import React, {FunctionComponent, ReactElement} from 'react'
+import {AdherenceAlert, AdherenceAlertCategory} from '@typedefs/types'
+import {theme} from '@style/theme'
 import dayjs from 'dayjs'
-import React, {FunctionComponent} from 'react'
+import {BorderedTableCell, StyledLink} from '../../widgets/StyledComponents'
+import {DateRange, PersonAddAlt, QuestionMark, TrendingDown} from '@mui/icons-material'
+import MoreVertIcon from '@mui/icons-material/MoreVert'
+import Menu from '@mui/material/Menu'
+import MenuItem from '@mui/material/MenuItem'
+import NotificationsNoneTwoToneIcon from '@mui/icons-material/NotificationsNoneTwoTone'
+import ParticipantService from '@services/participants.service'
+import PublishedWithChangesRoundedIcon from '@mui/icons-material/PublishedWithChangesRounded'
+import PersonPinCircleOutlinedIcon from '@mui/icons-material/PersonPinCircleOutlined'
 
 type AdherenceAlertsProps = {
   studyId: string
 }
 
-type AlertType = 'enrollment' | 'adherence' | 'upcoming'
+const CheckboxBox = styled(Box)(({theme}) => ({
+  padding: theme.spacing(1, 2),
+  display: 'flex',
+  mt: 0,
+  mb: 1,
+  background: '#F1F3F5',
+}))
 
-type AdhAlert = {
-  type: AlertType
-  participantId: string
-  timestamp: Date
-  text: string
-  done?: boolean
-}
-const _alerts: AdhAlert[] = [
-  {
-    type: 'adherence',
-    participantId: '323132',
-    timestamp: new Date(),
-    text: 'has adherence below 60%.',
-  },
-  {
-    type: 'upcoming',
-    participantId: '3231349',
-    timestamp: new Date(),
-    text: 'has and upcoming Burst in two weeks',
-  },
-
-  {
-    type: 'adherence',
-    timestamp: new Date(),
-    participantId: '3231321',
-    text: 'has adherence below 60%.',
-  },
-  {
-    type: 'upcoming',
-    participantId: '3231340',
-    timestamp: new Date(),
-    text: 'has and upcoming Burst in two weeks',
-  },
-  {
-    type: 'enrollment',
-    participantId: '32311432',
-    timestamp: new Date(),
-    text: 'has enrolled',
-  },
+const ALERT_CATEGORY: {label: string; value: AdherenceAlertCategory}[] = [
```

Comment:
I would have a single object that would be something like 
```
const ALERTS = {
new_enrollment: { label: 'New Enrollment', icon:  <PersonAddAlt />, text: 'has newly enrolled'},
low_adherence: { label: 'Adherence', icon:   <TrendingDown />, text: 'has adherence below [threshold]}`,
....
}
```

then you'd just do alertCategoryToLabel = ALERTS[whateveryourtypeis]. Same with images. for text you can do the same and then you can do `if(whateveryourtypeis ===  'low_adherence') {text = text.replace([threshold], JSON.parse(data).adherenceThreshold)}